src/sdk/analytics/utils/navigation.ts

```diff
@@ -0,0 +1,177 @@
+const ONE_YEAR_IN_MS = 31536000000
+const SESSION_ID = 'VtexRCSessionIdv7'
+const MAC_ID = 'VtexRCMacIdv7'
+
+/**
+ * Generates an Universally Unique Identifier (UUID) that is a 128-bit number, with the encoding in the
+ * format: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx. For this, the algorithm is based on version 4 of RFC4122,
+ * which can be better understood in the following documentation: https://www.ietf.org/rfc/rfc4122.txt.
+ *
+ */
+function uuidv4() {
+  const uuid = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+    const r = (Math.random() * 16) | 0
+    const v = c === 'x' ? r : (r & 0x3) | 0x8
+
+    return v.toString(16)
+  })
+
+  return uuid
+}
+
+/**
+ * Create or replace a cookie.
+ *
+ * @param {string} cookieId Cookie ID.
+ * @param {string} cookieValue Cookie's value.
+ * @param {number} expirationTime Time for the cookie to expire.
+ */
+function setVtexCookie(
+  cookieId: string,
+  cookieValue: string,
+  expirationTime: number
+) {
+  const formattedValue = encodeURIComponent(cookieValue)
+  const host = window.location.hostname
+  const expirationDate = new Date()
+
+  expirationDate.setTime(expirationDate.getTime() + expirationTime)
```

Comment:
I think this is more readable

```suggestion
  expirationDate.setTime(new Date().getTime() + expirationTime)
```

Reply:
Fixed